one-pager-maker/src/api/viewHistoryApi.ts

```diff
@@ -54,7 +54,15 @@ export const viewHistoryApi = {
             },
             onSuccess: async (history) => {
                 if (history === undefined) return;
-                await queryClient.refetchQueries({queryKey: [queryKeys.editHistoryId(history.id), queryKeys.editHistories]})
+                // queryClient.setQueryData([queryKeys.editHistoryId(history.id)], history);
```

Comment:
Did you forget to turn it off?

Reply:
I don't know what you mean, but I removed this comment `// queryClient.setQueryData([...`

Reply:
Did you forget to delete this comment? :)